src/api/actions/uploadMetadata.ts

```diff
@@ -18,6 +18,6 @@ export const uploadMetadata = async (
     throw Error(`Failed to upload metadata: ${e}`);
   }
 
-  const url = ipfsHashToUrl(response.ipfsHash);
+  const url = response.url;
```

Comment:
This will introduce a bug.

The metadata URL's that are to be stored in the smart contract should be of this format:
`ipfs://{IPFS HASH}` ie `ipfs://QmRL14Y5KsHF9BUoTkc9x1qkeCvTV2qESGypP8qRehTooS`

This is returning the public url: `https://ipfs.fleek.co/ipfs/QmRL14Y5KsHF9BUoTkc9x1qkeCvTV2qESGypP8qRehTooS`

Reply:
now returns `ipfsHashToUrl(response.hash)`